packages/venia-concept/webpack.config.js

```diff
@@ -121,12 +122,29 @@ module.exports = async env => {
         plugin => !browserPlugins.has(plugin.constructor.name)
     );
 
+    // remove browser-only module rules
+    serverConfig.module.rules = serverConfig.module.rules.map(rule => {
+        if (rule.oneOf) {
+            return {
+                ...rule,
+                oneOf: rule.oneOf.map(ruleConfig => ({
+                    ...ruleConfig,
+                    use: ruleConfig.use.filter(
+                        loaderConfig => loaderConfig.loader !== 'style-loader'
+                    )
+                }))
+            };
+        }
+
+        return rule;
+    });
+
     // add LimitChunkCountPlugin to avoid code splitting
     serverConfig.plugins.push(
         new LimitChunkCountPlugin({
             maxChunks: 1
         })
     );
 
-    return [config, serverConfig];
+    return config;
```

Comment:
Could only get one config to build at a time, complains about missing i18n generated file if this returns both configs.